site/content/en/latest/tasks/quickstart.md

```diff
@@ -15,8 +15,6 @@ __Note:__ Refer to the [Compatibility Matrix](/news/releases/matrix) for support
 __Note:__ In case your Kubernetes cluster does not have a LoadBalancer implementation, we recommend installing one
 so the `Gateway` resource has an Address associated with it. We recommend using [MetalLB](https://metallb.universe.tf/installation/).
 
-__Note:__ For Mac user, you need install and run [Docker Mac Net Connect](https://github.com/chipmk/docker-mac-net-connect) to make the Docker network work.
```

Comment:
why remove this?

Reply:
its distracting for an end user, its tied to mac and docker and kind, for e.g. on mac, i use docker desktop and this isnt need, this note is needed in developer docs, when running e2e on kind on mac